scripts/opt/backup-loop.sh

```diff
@@ -230,6 +231,15 @@ tar() {
     find "${DEST_DIR}" -maxdepth 1 -name "*.${backup_extension}" -mtime "+${PRUNE_BACKUPS_DAYS}" "${@}"
   }
 
+  _find_extra_backups() {
+  find "${DEST_DIR}" -maxdepth 1 -name "*.${backup_extension}" -print0 | \
+    xargs -0 stat --format '%Y %n' | \
+    sort -n | \
+    awk '{print $2}' | \
+    tail -n +$((PRUNE_BACKUPS_COUNT + 1)) | \
+    tr '\n' '\0'
```

Comment:
I think this could be simplified by using `ls -t` to sort by time:

```suggestion
  find "${DEST_DIR}" -maxdepth 1 -name "*.${backup_extension}" -exec ls -t {} \+ | \
    tail -n +$((PRUNE_BACKUPS_COUNT + 1))
```